notebooks/thunder_as_torch_compile_backend_tutorial.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p>If we make the change proposed below to have an actual <code>thunderfx</code> function, then I think the title of this document could be:</p><p> </p><p><code>"Hello, World!" ThunderFX</code></p><p> </p><p>Then the introduction can say, like you have now, that this tutorial will show how to use ThunderFX to optimize PyTorch programs, especially PyTorch CUDA programs. The idea that ThunderFX uses torch.compile internally can come later, in your (very good) discussion of FX graphs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>